packages/chat/commands/cover-traffic.ts

```diff
@@ -0,0 +1,73 @@
+import { encode, decode } from 'rlp'
+import type HoprCoreConnector from '@hoprnet/hopr-core-connector-interface'
+import type Hopr from '@hoprnet/hopr-core'
+import { AbstractCommand } from './abstractCommand'
+
+const INTERVAL = 1000
+
+export class CoverTraffic extends AbstractCommand {
```

Comment:
can we add a short description of what this command does?

Reply:
ok